controller: Default
button_move_deadman_switch: 6
button_rotate_deadman_switch: 4
button_turbo: 7
axis_move_x: 1
axis_move_y: 0
axis_move_z: 2
axis_rotate_x: 3
axis_rotate_y: 1
axis_rotate_z: 0

